package.json

```diff
@@ -76,5 +76,8 @@
     "env": {
       "node": true
     }
+  },
+  "dependencies": {
+    "typescript": "^5.3.3"
```

Comment:
Move this to `devDependencies` (dependencies that are onoly used during development).

The `dependencies` are reserved for runtime dependencies - libraries that are needed when the final code runs.

In this project we should aim to have 0 `dependencies` in order to keep the final script small.